volatility/framework/symbols/windows/__init__.py

```diff
@@ -14,10 +14,12 @@ def __init__(self, *args, **kwargs) -> None:
 
         # Set-up windows specific types
         self.set_type_class('_ETHREAD', extensions.ETHREAD)
+        self.set_type_class('_KTHREAD', extensions.KTHREAD)
```

Comment:
These are the changes already from a previous merge, so this branch will need rebasing I'm afraid.  5:(